tests/fixtures/optimizer/eliminate_join_marks.sql

```diff
@@ -0,0 +1,41 @@
+ 
+# title: Constant in join  
+# dialect: oracle
+SELECT T1.d, T2.c FROM T1, T2 WHERE T1.x = T2.x (+) and T2.y (+) > 5;
+SELECT T1.d, T2.c FROM T1 LEFT JOIN T2 ON T1.x = T2.x AND T2.y > 5;
+  
+# title: Null in Join Predicte  
+# dialect: oracle
+SELECT T1.d, T2.c FROM T1, T2 WHERE T1.x = T2.x (+) and T2.y (+) IS NULL;
+SELECT T1.d, T2.c FROM T1 LEFT JOIN T2 ON T1.x = T2.x AND T2.y IS NULL;
+  
+# title: Null in Join Predicte 2  
+# dialect: oracle
+SELECT T1.d, T2.c FROM T1, T2 WHERE T1.x = T2.x (+) and T2.y IS NULL;
+SELECT T1.d, T2.c FROM T1 LEFT JOIN T2 ON T1.x = T2.x WHERE T2.y IS NULL;
+  
+# title: Additional where clause  
+# dialect: oracle
+SELECT T1.d, T2.c FROM T1, T2 WHERE T1.x = T2.x (+) and T1.Z > 4;
+SELECT T1.d, T2.c FROM T1 LEFT JOIN T2 ON T1.x = T2.x WHERE T1.Z > 4;
+  
+# title: various select tests 1  
```

Comment:
These titles are not very helpful, let's remove them.

Reply:
Or use more descriptive names :)

Reply:
This file is now removed as the test have moved to the oracle dialect test suite.